ckanext/cfpb_extrafields/validators.py

```diff
@@ -6,11 +6,6 @@ def Invalid(message):
     import ckan.plugins.toolkit as tk
     raise tk.Invalid(message)
 
-def required_field(value):
```

Comment:
oh good, I didn't like this method anyway.  Does that mean we found a built-in substitute for it?